chsdi/templates/htmlpopup/einschraenkungen_drohnen.mako

```diff
@@ -31,6 +32,16 @@
     zone_name = 'zone_name_%s' % lang
     zone_restriction = 'zone_restriction_%s' % lang
     zone_message = 'zone_message_%s' % lang
+    auth_url = 'auth_url_%s' % lang
```

Comment:
does the db guarantee that NULL values are not allowed here?

Reply:
this is tested later I think with:

' / '.join(c['attributes'][auth_url]) if c['attributes'][auth_url] else '-'

Reply:
@faselm yes, indeed - in the extended part it is being tested. In the regular tooltip not.

Reply:
I've just added it